src/main/httpCommandServer.ts

```diff
@@ -118,32 +127,87 @@ export class HttpCommandServer {
     return this.dispatchTable[version]?.[method]?.[commandName];
   }
 
-  listSettings(request: http.IncomingMessage, response: http.ServerResponse) {
-    const settings = this.commandWorker?.getSettings();
+  listSettings(request: http.IncomingMessage, response: http.ServerResponse): Promise<void> {
+    const settings = this.commandWorker.getSettings();
 
     if (settings) {
+      console.log('listSettings: succeeded 200');
       response.writeHead(200, { 'Content-Type': 'text/plain' });
       response.write(settings);
     } else {
+      console.log('listSettings: failed 200');
       response.writeHead(404, { 'Content-Type': 'text/plain' });
       response.write('No settings found');
     }
+
+    return Promise.resolve();
+  }
+
+  /**
+   * Handle `PUT /vx/set` requests.
```

Comment:
This might be easier to read as:
```suggestion
   * Handle `PUT /v?/set` requests.
```

Reply:
sure